### DISCUSSION TITLE
Full Issue details do no display when issue accessed from Project

### DISCUSSION THREAD

**Discussion by jehtee** [community/community#64890] [New to GitHub]
Because GH doesn't have Epics, and task lists remain in a restricted beta, I've had to hack Issues to act as Epics.  Part of this is that the individual issues need to display backlinks to the Epic that references them.  

But when I access an issue from within a Project, only the comments display, none of the other activity.

How do I get ALL of an Issue's content to display when accessed from within Projects?  Or at least how do I get the backlinks to display?

Or is there a better way to make an Epic than to repurpose Issues?

**Comment by lab596** [comment]
Markdown Links:
If you're using Markdown in your Issues, you can create links manually within the issue description to reference the "Epic" issue. For example, you could use [Epic Issue](link-to-epic) to create a hyperlink to the Epic.

**Reply by jehtee** [reply]
But this should not require a hack.  The data is available, its just not being presented.  Viewing an issue from within or outside of a Project should show all of the same issues properties.

(Also, you link goes to a 404)
